tests/test_bibliotheca.py

```diff
@@ -1246,6 +1246,16 @@ def test_contributors_for_string(cls):
             "Baum, Frank L. (Frank Lyell)")
         assert "Baum, Frank L." == author.sort_name
 
+        # Contributors may have two levels of entity reference escaping,
+        # one of which will have already been handled by the initial parse.
+        # So, we'll test zero and one escapings here.
+        authors = list(ItemListParser.contributors_from_string(
+            u'Raji Codell, Esmé; Raji Codell, Esm&#233;'))
```

Comment:
In normal operation, `ItemListParser.contributors_from_string` receives Unicode strings in py2, regardless of what the surround tests imply.